src/support/thread_group.c

```diff
@@ -23,6 +23,7 @@ __thread_run(void *arg)
 	session = thread->session;
 
 	for (;;) {
+		WT_ERR(WT_SESSION_CHECK_PANIC(session));
```

Comment:
I don't understand why we're checking panic here -- at best I don't think it makes anything better. There's some related discussion in WT-3245.